src/components/claiming/ClaimingInfo.vue

```diff
@@ -106,24 +108,31 @@ export default {
       display: flex;
       padding: 16px;
       .data {
+        flex: 1;
         margin-right: 24px;
         font-size: 16px;
         .currency {
           font-size: 12px;
         }
       }
+      .amount {
+        flex-basis: 100%;
+      }
       .date {
+        flex-basis: 100%;
+        text-align: center;
```

Comment:
<details>
<summary>Joke</summary>

:rocket: [Click here to go to the future](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout) :rocket: 
:rocket: [Click here to go to the past](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table)  :rocket: 
</details>

I think this would be easily developed with `table` > `grid` > `flex`. We have to decide wisely next time.